crates/ruff_python_formatter/resources/test/fixtures/ruff/skip_magic_trailing_comma.options.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "magic_trailing_comma": "Respect"
+  },
+  {
+    "magic_trailing_comma": "Ignore"
```

Comment:
Clever, I like it. (Should these be lowercase?)

Reply:
It would be nice if we could configure the use of `kebabCase` globally rather than on an enum to enum basis.